btk/sampling_functions.py

```diff
@@ -55,18 +55,20 @@ def compatible_catalogs(self):
 class DefaultSampling(SamplingFunction):
     """Default sampling function used for producing blend tables."""
 
-    def __init__(self, max_number=2, stamp_size=24.0, maxshift=None):
+    def __init__(self, max_number=2, stamp_size=24.0, maxshift=None, rng=np.random.default_rng()):
```

Comment:
Having `default_rng()` without specified seed as the default value of a keyword does not seem a good idea to me.
I'd rather have `None` and then do something about it.